### PR TITLE
[15.0][MIG] base_comment_template: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Superseed: https://github.com/OCA/reporting-engine/pull/619

Migration to 15.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT35696

### CODE REVIEW

File: base_comment_template/wizard/base_comment_template_preview_views.xml [hunk 1]
```diff
@@ -1,80 +1,67 @@
...
-                    </div>
-                    <p>Choose a language:
-                        <field name="lang" class="oe_inline ml8" />
-                    </p>
-                    <p>Choose an engine:
-                        <field name="engine" class="oe_inline ml8" />
-                    </p>
-                    <label for="body" string="Base Comment content" />
-                    <hr />
+    <!-- SMS Template Preview -->
```
Comment (pedrobaeza) [inline base_comment_template/wizard/base_comment_template_preview_views.xml]: This has no sense

File: base_comment_template/wizard/base_comment_template_preview.py [hunk 2]
```diff
@@ -41,21 +39,25 @@ def default_get(self, fields):
...
     engine = fields.Selection(
-        [("jinja", "Jinja"), ("qweb", "QWeb")],
+        [
+            ("inline_template", "Inline template"),
+            ("qweb", "QWeb"),
+            ("qweb_view", "QWeb view"),
+        ],
         string="Template Preview Engine",
-        default="jinja",
+        default="inline_template",
```
Comment (pedrobaeza) [inline base_comment_template/wizard/base_comment_template_preview.py:50]: A migration script is needed to convert existing values to this new one.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please add a commit on top of the previous one with your changes instead of blending them with the existing.

**Review by hailangvn — APPROVED** [review]
Functional test ok. Thanks for the work.

**Comment by marylla** [comment]
@victoralmau Can you please update your PR as Pedro suggested. Thank you! :)

**Comment by victoralmau** [comment]
The commits are already separated and I have added the migration script to convert the text of templates.

**Comment by marylla** [comment]
@pedrobaeza Can you review again please?

**Review by pedrobaeza — COMMENTED** [review]
Missed `jinja` to `inline_template` value mapping

**Comment by victoralmau** [comment]
> Missed `jinja` to `inline_template` value mapping

Sorry, I don't understand exactly what you mean.

The engine change from `jinja` to `inline_template` is only used in wizard to rendering the comment.

**Review by pedrobaeza — APPROVED** [review]
OK, seen

**Pull request merged**